hooks/read-package-hook/src/createVersionsOverrider.ts

```diff
@@ -61,25 +58,28 @@ interface VersionOverride extends VersionOverrideBase {
   localTarget?: LocalTarget
 }
 
-type VersionOverrideWithParent = VersionOverride & Required<Pick<VersionOverride, 'parentPkg'>>
+interface VersionOverrideWithParent extends VersionOverride {
+  parentPkg: PackageSelector
+}
```

Comment:
The TypeScript compiler works faster with `interface extends` than with object intersection.